openai/README.md

```diff
@@ -29,7 +29,102 @@ Get cost estimation, prompt and completion sampling, error tracking, performance
 <!-- web-ui. Make sure to update the markdown / code there to see any changes take -->
 <!-- effect on the tile. -->
 
-#### APM: Get Usage Metrics for Python and Node.js Applications
+<!-- xxz tab xxx -->
+<!-- xxx tab "Python" xxx -->
+
+**Note**: This setup method does not collect `openai.api.usage.*` metrics. To collect these metrics, also follow the API key setup instructions.
+
+### Installation
+
+#### LLM Observability: Get end-to-end visibility into your LLM application's calls to OpenAI
+You can enable LLM Observability in different environments. Follow the appropriate setup based on your scenario:
+
+##### If you do not have the Datadog Agent:
+1. Install the `ddtrace` package:
+
+```shell
+  pip install ddtrace
+```
```

Comment:
Hey @yahya-mouman, these changes look great! As a minor suggestion, when you're nesting a code block inside a step, I find it helpful to indent the code block so it's in line with the rest of the step content. That way, it's much easier for your users to scan through your procedure and see where different steps begin and end. Here's an example:

```suggestion
1. Install the `ddtrace` package:

   ```shell
     pip install ddtrace
   ```
```